### DISCUSSION TITLE
MEETING: esites Working Group - 21st March @3:30-4pm

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#210] [MEETINGS]
# Date: 21st March 2022 @3:30-4pm

# Agenda: 
**> Reminder: esites [Agreed Data Entry Standards](https://github.com/fieldmuseum/EMu-Documentation/wiki/ESTABLISHED-STANDARDS#esites) and [Agreed Country Values](https://github.com/fieldmuseum/EMu-Documentation/issues?q=is%3Aissue+is%3Aopen+label%3Aesites+label%3ASTANDARD+).**

**> Current State of New Zealand Records in EMu**
> > 5190 Records with PolPD1 = New Zealand
Summary Report - Political Rank
1	99	city
2	3	Country
3	2	County
4	11	District
5	11	pd2
6	66	pd3
7	4919	Precise Locality
8	16	Region
9	63	(empty)
	5190	Total

> > 5 Records contain LocIslandMainName = New Zealand
> > > Aotearoa (New Zealand Archipelago)
> > > Aotearoa (New Zealand Archipelago) or Samoan Islands
> > > Aotearoa (New Zealand Archipelago) or Solomon Islands Archipelago
> > > Aotearoa (New Zealand Archipelago), Rekohu (Chatham Islands)
> > > Cook Islands [New Zealand]

**> New Zealand Updates from Botany and Anthro on their conversations**
>> Maori co-equal placenames: How to get them in properly

**> New Zealand Open github issues**
> > Question 16 [2022]: Sorting out New Zealand [RS] https://github.com/fieldmuseum/EMu-Documentation/discussions/171
> > Question 24 [2022]: Sites questions [WG, CP] https://github.com/fieldmuseum/EMu-Documentation/discussions/212
> > QUESTION 25 [2022]: NZ site opinions [WCG] https://github.com/fieldmuseum/EMu-Documentation/discussions/214

**_TASKS:_**
1. Cleanup pd2/pd3/Empty
2. Document New Zealand Decisions (#214)
3. 

**> Decision process moving forward. How do we want it to work?**
> > Question 23 [2022]: Is there a way to have more discussion about what sites records are being modified and when? [CP; WG] 

_Generated from EMu user meeting topics #209 #212 #214_

**Comment by rondlg** [comment]
# MEETING NOTES:
**RS, SG, WG, TS, PM, SM, KGJ, JJ, KW, ZK, MT, AF, BM, CP, CM, KS**

- What did Botany and Anthro decide when they met:
- - Each region will have an island as a parent (pd2 can clean up to that)
- - - https://en.wikipedia.org/wiki/Regions_of_New_Zealand
- Which pd2/pd3 list are we using as the standard?
**- [WG, CP] DECISION: The wikipedia lists** 
- - 16 official region records (PD2s) https://en.wikipedia.org/wiki/Regions_of_New_Zealand
- - PD3 - territorial authority  https://en.wikipedia.org/wiki/Territorial_authorities_of_New_Zealand 

- Ecological regions and districts are a different way to do it but Bot and Ant didn’t feel the need to document those - do other people need/want that. **DECISION: Not at this time**
- - They don’t overlap with the Political Regions
- - Ben: What are the eco districts?
- - WG: Referencing an official list [which?]  (not N/S island.)
- - - https://www.doc.govt.nz/Documents/science-and-technical/Ecoregions1.pdf 
- - - These could be Special Geographic regions
- - AF - if not a geopolitical region, wouldn’t usually put on a label [Mammals only has <10 specimens tho]
- - BM - likewise, ~40 birds specimens, but some = Chatham Isl.

- Chatham Island is a pd2 with a rank 

- “Cook Islands [New Zealand]” Sites irn 630864
- - Child sites IZ in Island Hierarchy
- - CP - NZ = protectorate of Cook Islands -
- - Set NZ as Sovereignty attachment on ‘Sites’ tab of current Island-children
- - This comes from the IZ records. _TASK: KGJ will update to put NZ in sovereignty_ 

- If someone searches for NZ on websites, will sovereignty be searched as well as country?
- - Do all depts/collections want that? (Likewise - North America / Hawaii searches)
- - CP - we wouldn’t want Cook Islands / Sovereignty to come back on web / for users searching Country = NZ
- - KGJ - likewise + Splitting out helps 

- RB: Insects only uses PD2 so far, so wouldn’t use ecoregions

- and pd3 = https://en.wikipedia.org/wiki/Territorial_authorities_of_New_Zealand 

- [CP] Can we get West Indies back into the Grenada hierarchy?
- - irn 629294 = island Grenada
- - irn 21091 = country? Grenada
- _TASK: Update the island record or Grenada to include its political parent_

- **DECISION: Agreed to use Pacific on ALL Oceania localities (from the Continent->Precise Locality)**
- - Thumbs up: RB, ZK, TS, CP, AF, WG
- - _TASK: Blanket update Cultural on all existing record._
- - _TASK: Make sure the cultural tab is switched on all groups._

- Using alternate Maori place names (co-equal official terms)
- - **DECISION: No objections to using alternatives**

- Contributing to the EMu Documentation github repo
- _TASK: Training / Github for those without accunts yet:_
- - KS, KJG, CP, SM, BM

- RSB - Could the repositories of these countries with co-equal names be posted on a searchable GitHub topic?

- Workflow/Process for working on sites that are shared between groups
- - Communications
- - - Add standing item to EMu meetings for data cleanup/standards
- - - Use EMu-user email to let people know that a standard has been agreed 
- - How to know who else might be using a sites record that you want to cleanup?
- - - Check attachments, there are a number of places that you have to check for attachments: Catalogue; collectionevents; other sites
- - - _TASK: Document how Tech currently checks these things._
- - It would be great to do a big sweep and systematically clean all sites but not practical. So have to do it case by case.

**Comment by rondlg** [comment]
DECISION on [dual place names](https://github.com/fieldmuseum/EMu-Documentation/wiki/admin-ESTABLISHED-STANDARDS#how-to-document-dual-place-names) is documented in the esites section of[ ESTABLISHED STANDARDS page](https://github.com/fieldmuseum/EMu-Documentation/wiki/admin-ESTABLISHED-STANDARDS).

**Comment by rondlg** [comment]
DECISION [Cultural vs ISO Continent issue ](https://github.com/fieldmuseum/EMu-Documentation/issues/2)updated to reflect decision that all areas will include Pacific in the Cultural Region field of sites classified as PD0=Oceania.

**Reply by fmjjones** [reply]
All Oceania sites records now have Cultural Region = Pacific.

**Comment by rondlg** [comment]
Grenada island record has its political parent added during the meeting

**Comment by rondlg** [comment]
New Zealand data cleanup issue created: https://github.com/fieldmuseum/EMu-Documentation/issues/571
